### PR TITLE
update CHANGELOG

### PR DESCRIPTION
**Pull request by vlad-perevezentsev**

### CODE REVIEW

**Comment by vlad-perevezentsev** [comment]
What should i write for this commit?
https://github.com/IntelPython/numba-dppy/commit/0105e830a42829ea7206cf4d87d9f9d9246253f2

File: numba_dppy/CHANGE_LOG.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+### Removed
+- Use of cffi.
+- The old backup file.
+
+## NUMBA Version 0.48.0 + DPPY Version 0.3.0 (June 29, 2020)
+
+This release includes:
+- Caching of dppy.kernels which will improve performance.
+- Addition of support for Intel Advisor which will help in profiling applications.
```
Comment (PokhodenkoSA) [inline numba_dppy/CHANGE_LOG.md:24]: Where did you take it?

Comment (PokhodenkoSA) [reply numba_dppy/CHANGE_LOG.md:24]: I see. From file numba_dppy/CHANGE_LOG.

**Comment by diptorupd** [comment]
> 
> 
> What should i write for this commit?
> [0105e83](https://github.com/IntelPython/numba-dppy/commit/0105e830a42829ea7206cf4d87d9f9d9246253f2)

@reazulhoque please add a one-two line Chalgelog entry for the rewrite pass. Would be good to focus on actual change (`lower_builtin` to `overload`) and the new functions that go added such as `ndarray.sum`.

**Comment by PokhodenkoSA** [comment]
File name should be CHANGELOG.md

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+- numba-dppy is a standalone package now.
+- Offload diagnostics.
+- Controllable fallback.
+- Add flags to generate debug symbols.
+- Implementation of `np.linalg.eig`.
+
+### Changed
+- Rename `DPPL` to `DPPY`.
+- `DPPLTestCase` replaced with `unittest.TestCase`.
+- All tests and examples use `with device_context`.
```
Comment (diptorupd) [inline CHANGELOG.md:22]: - The `dpnp` integration to support NumPy functions on SYCL devices now uses Numba's `@overload` functionality.

Comment (PokhodenkoSA) [reply CHANGELOG.md:24]: Done

**Review by PokhodenkoSA — APPROVED** [review]
Need some items from @reazulhoque. Could be merged then.

**Pull request merged**